src/webgpu/shader/validation/expression/call/builtin/sinh.spec.ts

```diff
@@ -47,7 +52,10 @@ Validates that constant evaluation and override evaluation of ${builtin}() rejec
   })
   .fn(t => {
     const type = kValuesTypes[t.params.type];
-    const expectedResult = isRepresentable(Math.sinh(t.params.value), elementType(type));
+    const expectedResult = isRepresentable(
+      Math.sinh(Number(t.params.value)),
+      elementType(type).kind === 'abstract-int' ? TypeAbstractFloat : elementType(type)
```

Comment:
Is this right?

Reply:
ditto, found a few other examples and fixed them

Reply:
ditto to my correction above